### PR TITLE
[Master] Update compiler plugin to allow annotations

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose

2201.7.x https://github.com/ballerina-platform/module-ballerinax-nats/pull/770

## Checklist
- [x] Linked to an issue
- [x] Updated the changelog
- [x] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 # Change Log
 This file contains all the notable changes done to the Ballerina NATS package through the releases.
 
-[Unreleased]
+## [2.9.1] - 2023-08-21
 
 ### Changed
-- [[#4237] Exit the Listener When Panic Occurred](https://github.com/ballerina-platform/ballerina-standard-library/issues/4237)
 
+- [[#4733] Changed disallowing service level annotations in the compiler plugin](https://github.com/ballerina-platform/ballerina-standard-library/issues/4733)
```
Comment (ThisaruGuruge) [inline changelog.md:8]: Is this release version correct?

Comment (aashikam) [reply changelog.md:8]: What do you mean 😮
I just released https://github.com/ballerina-platform/module-ballerinax-nats/releases/tag/v2.9.1

Comment (ThisaruGuruge) [reply changelog.md:8]: There are no records for the previous releases. That's what I meant. In a given branch, the changelog should include all the changes released from that branch. In this case, the release was done using another branch. So basically, on this branch, these are not released. It is incorrect to mention them as released.

Comment (aashikam) [reply changelog.md]: ooo didn't know 😮
Update 02b06108f836229bd064de928937cc9e2a7f7b66

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**